compiler/src/iree/compiler/Dialect/LinalgExt/IR/AggregatedOpInterfaceImpl.cpp

```diff
@@ -178,6 +178,28 @@ static Value computeMatmul(OpBuilder &builder, Location loc, AffineMap lhsMap,
   return genericOp.getResult(0);
 }
 
+static Value applyRegion(OpBuilder &builder, Location loc, Region &region,
```

Comment:
nit: rename to applyPostQKMatmulElementwise or something more descriptive.

Reply:
Updated.